### PR TITLE
add yellowbrick python package

### PR DESCRIPTION
**Pull request by melonhead901**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by melonhead901** [comment]
FYI I created [a PR](https://github.com/DistrictDataLabs/yellowbrick/pull/1092) in the source repo to include license file in the source distrbiution

File: yellowbrick/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - cycler >=0.10.0
+    - matplotlib >=2.0.2,!=3.0.0
```
Comment (scopatz) [inline yellowbrick/meta.yaml:24]: This should be `matplotlib-base`

File: yellowbrick/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3edddb4a16ddd0586169fccd0645b2fa78e266c40968f872753909250ac6826c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline yellowbrick/meta.yaml:14]: I think this can be ``noarch``
```suggestion
  noarch: python
```

File: yellowbrick/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - yellowbrick.features
+    - yellowbrick.gridsearch
+    - yellowbrick.model_selection
+    - yellowbrick.regressor
+    - yellowbrick.style
+    - yellowbrick.target
+    - yellowbrick.text
+    - yellowbrick.utils
+  requires:
+    - pytest
```
Comment (marcelotrevisani) [inline yellowbrick/meta.yaml:48]: As you are just testing the imports you do not need ``pytest`` here, right?
```suggestion
```

File: yellowbrick/meta.yaml (hunk 1 above)
Comment (melonhead901) [reply yellowbrick/meta.yaml:24]: Done

File: yellowbrick/meta.yaml (hunk 2 above)
Comment (melonhead901) [reply yellowbrick/meta.yaml]: Removing the script command made the build fail, but I'll leave noarch

File: yellowbrick/meta.yaml (hunk 3 above)
Comment (melonhead901) [reply yellowbrick/meta.yaml]: Removing this also made the build fail.

Comment (chrisburr) [reply yellowbrick/meta.yaml]: Can you remove this again, I think the failures were related to the other changes

**Pull request merged**